tutorials/axios/user-agent.md

```diff
@@ -0,0 +1,13 @@
+[Axios](https://github.com/axios/axios) has a `User-Agent` property which allows you to to identify
+the type of device making the request to the server so it knows what data to send back.
```

Comment:
Not necessarily true, it is very rare that servers send back different responses based on User-Agent. Most of the time it's just for analytics.